### PR TITLE
[🚧 WIP] Test GitHub Actions running on every PR

### PR DESCRIPTION
**Pull request by amyrlam**
## :pushpin: Summary

Before, tests didn't run if you didn't edit files in `ember-flight-icons/`. This was confusing to me, and would probably be confusing to external contributors.

### :speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

Examples: 
- issue (ux,non-blocking): These buttons should be red, but let's handle this in a follow-up.
- suggestion (non-blocking): Let's change this wording to make it easier to understand.
- issue (blocking): We shouldn't introduce this kind of tech debt; let's pair and resolve the issue in a more sustainable way.

## :link: External Links

Resolves: https://github.com/hashicorp/flight/issues/102

### CODE REVIEW

**Comment by amyrlam** [comment]
i thought this would be really quick, checked out https://github.com/jelhan/create-github-actions-setup-for-ember-addon, i see why it's not that simple now. a bit different from travis!  

maybe an intermediate step could be to add linting to `flight-icons/`, which seems like it would be a separate github actions workflow. although not sure if that's overkill. will revisit this later

**Comment by amyrlam** [comment]
for context: i was looking for low-hanging fruit to pick off before i'm out

tests didn't fire at all

i think there needs to be a separate "action" for the top level repo, since this one was created (maybe?) from the travis to github actions file automated thing? i started to look at their docs and #tech-github-actions on slack

File: .github/workflows/ci-ember-flight-icons.yml [hunk 1]
```diff
@@ -30,18 +30,11 @@
 
 name: CI - Ember Flight Icons
 
-defaults:
-  run:
-    working-directory: ember-flight-icons
```
Comment (Dhaulagiri) [inline .github/workflows/ci-ember-flight-icons.yml:35]: Took another pass at this in https://github.com/hashicorp/flight/pull/153. I think removing this line might have caused problems in this PR as this is still required for the tests to work correctly

Comment (amyrlam) [reply .github/workflows/ci-ember-flight-icons.yml:35]: ah gotcha - thank you for the follow-up!

**Pull request closed without merging**